report_thesis/src/sections/problem_definition.tex

```diff
@@ -1,23 +1,60 @@
+    \item \textbf{Matrix $D_{t \times l \times w}$}: Derived from matrix $C$ by averaging the intensities across shots to provide a clearer signal for each location and wavelength:
+    \[
+    D[t, l, w] = \frac{1}{|S|} \sum_{s \in S} C[t, l, s, w].
+    \]
+    \item \textbf{Matrix $E_{t \times l \times w}$}: The result of $D$ processed by applying wavelength-specific masks, setting intensities to zero in masking ranges to focus on relevant spectral features.
+\end{itemize}
 
-\gls{libs} uses a laser to ablate surface material and generate a plasma plume, which emits light captured by spectrometers. 
-The resulting spectra consist of emission lines associated with the concentration of specific elements, serving as a multi-dimensional fingerprint of the sample's elemental composition~\cite{cleggRecalibrationMarsScience2017}.
+The outputs of our models are the quantified chemical compositions of geological samples. These are primarily the concentrations of major elements, represented in weight percentage. While trace elements are also present, our current analysis does not quantify these.
+Our goal is to construct a mapping function $\mathcal{F}: \mathbb{R}^N \rightarrow \mathbb{R}^O$, where $N$ represents the dimensionality of the processed \gls{libs} signals and $O$ represents the number of target oxides. This function maps a processed \gls{libs} signal vector $\mathbf{x} \in \mathbb{R}^N$ to a vector $\mathbf{v} \in \mathbb{R}^O$ of estimated oxide concentrations:
+\[
+\mathbf{v} = \mathcal{F}(\mathbf{x}).
+\]
+
+The primary metric used to evaluate the performance of our computational models is the \gls{rmse}. \gls{rmse} is calculated using the formula:
+\[
+\text{RMSE} = \sqrt{\frac{1}{n} \sum_{i=1}^{n} (\mathbf{v}_i - \hat{\mathbf{v}}_i)^2}
+\]
+where \( \mathbf{v}_i \) is the vector of actual oxide concentrations for the \( i \)-th sample, \( \hat{\mathbf{v}}_i \) is the corresponding vector of predicted oxide concentrations, and \( n \) is the total number of samples. This measure quantifies the average magnitude of the prediction error across all predicted values, providing a clear indication of model accuracy in quantifying chemical compositions.
+
+Our evaluation involves a combination of methodologies. Initially, the model is validated using a train/test split, where it is trained on a subset of data and tested on a separate subset to assess its predictive performance. Further, we compare the performance of our models against a baseline model previously established in our research \cite{p9_paper}, as well as against chemical analysis methods and other computational models documented in existing literature. This comprehensive comparison helps to underscore the improvements and benefits our approach offers over traditional methods and prior computational approaches.
```

Comment:
I dont think this part fits here. I would expect the problem definition to be about what the problem is and why its important in general terms. This feels more like coming up with a solution. Well written though.

Reply:
Removed!